pages/box-office/_performanceId/sell/complete.vue

```diff
@@ -51,10 +61,25 @@ export default {
   data() {
     return {
       checkedIn: false,
+      currentTime: new Date(),
     }
   },
+  computed: {
+    performanceDoorsDiffMinutes() {
+      return (
+        (Date.parse(this.booking.performance.doorsOpen) - this.currentTime) /
```

Comment:
Could use the Luxon DateTime library here (already installed) - take a look at https://github.com/BristolSTA/uobtheatre-web/blob/develop/pages/publicity-screen/_venueSlugs/index.vue#L146 and https://moment.github.io/luxon/#/math?id=diffs